src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -48,12 +51,16 @@ class AdminModuleDataProvider implements ModuleInterface
 
     const _DAY_IN_SECONDS_ = 86400; /* Cache for One Day */
 
+    protected $module_actions = array('install', 'uninstall', 'enable', 'disable', 'enable_mobile', 'disable_mobile', 'reset', 'upgrade');
```

Comment:
Please change `$module_actions` to `$moduleActions`

Reply:
Please add a PhpDoc header on your new properties.

It is important that each property describes:
- A short summary of what it holds / its purpose
- Type

Documenting properties is important because
- It enables autocompletion by IDEs
- It eases code understanding

Example:
```php
   /**
     * Query string parameters ($_GET).
     *
     * @var \Symfony\Component\HttpFoundation\ParameterBag
     */
    public $query;
```

You can find out more about PhpDoc by reading the [proposed PSR-5 standard](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md).